### DISCUSSION TITLE
Types of Repositories

### DISCUSSION THREAD

**Discussion by bkelly7769** [community/community#21875] [New to GitHub]
<p>I am having a difficult time solidifying some concepts in my mind. Git uses distributed repositories. Is there a single one that is used as the, for lack of a better name, primary repository? Is there one repository that, for example, only Sally can update and this is the one used for our final product build? Does it have a special name?</p>
<p>I am presuming that the answer is no, there is no formal primary repository, all are equal. However, I presume, pretty much all git users/communities declare one repository to be the primary one. This is a procedural concept and not enforced by any ability within git itself.</p>
<p>Do I need a redirect?</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/bkelly7769">@bkelly7769</a>,</p>
<p>You are absolutely correect in your analysis there. Git has no concept of a primary repo, this is purely a social convention. GitHub was built for developers to make it easy to *have* such a de-facto primary repository, but nothing in Git requires that such a primary exists.</p>

**Comment by bkelly7769** [comment]
<p>Got it, thank you for the reply.</p>
